### ISSUE TITLE
Data Structures in C++.

### ISSUE THREAD

**Issue by Jashandeep2020** [sukritishah15/DS-Algo-Point#23]
## 🚀 Feature

Want to add some data structures code in the c-plus-plus folder like Linked List and Stack.

### Have you read the Contribution Guidelines?

Yes.

## Pitch

Only one code is present in c-plus-plus folder, so it would be better to have some more data structures implementation.

**Comment by Jashandeep2020** [comment]
Want to make contributions under Hacktoberfest.

**Comment by codewithnick** [comment]
i also can do stack in python

**Comment by codewithnick** [comment]
please assign me if a issue is opened for the same

**Comment by lostgirljourney** [comment]
@sukritishah15 di, I can do Implementation of Singly Linked List in C.

**Comment by Jashandeep2020** [comment]
Can you please assign this issue to me, for the implementation in c++?

**Comment by sukritishah15** [comment]
@Jashandeep2020 - Singly LL and Stack in C++
@codewithnick - Stack in Python
@lostgirljourney - Singly Linked List in C

**Comment by Anni852** [comment]
Can I do Doubly LL in C and C++

**Comment by sukritishah15** [comment]
Doubly LL
C - @Anni852
C++ - @Anni852

**Comment by patricktrp** [comment]
I can do Doubly LL in Python

**Comment by patricktrp** [comment]
I'd do a Singly Linked List in Python

**Comment by Karthik-Shenoy** [comment]
I'd like to do Singly Linked List, Doubly Linked List, Circular Singly Linked List, Circular Doubly Linked List and Queue in python
can you please assign me to solve these issues

**Comment by sukritishah15** [comment]
@patricktrp - Singly Linked List in Python
@Karthik-Shenoy - Doubly Linked List, Circular Singly Linked List, Circular Doubly Linked List and Queue in Python

**Comment by scshiv29-dev** [comment]
assign me stack and queue in java please @sukritishah15

**Comment by osomat123** [comment]
Hello, could you please assign me with Circular Singly Linked List, Circular Doubly Linked List, Stack, Queue in C?

**Comment by Rahulraj31** [comment]
want to make contribution for Hacktoberfest pls assign me singly/ doubly linked in list in C++ or Python

**Comment by aditik03** [comment]
assign me task circular singly/doubly linked list, stack, and queue to implement in c.

**Comment by mukulpathania96** [comment]
I can implement singly LL and doubly LL in java. 
Please assign me the task.

**Comment by APOORVANAGAR-2002** [comment]
Can you assign me queue in C++?

**Comment by AnanyaNagar** [comment]
Can you assign circular singly linked list in c++

**Comment by sukritishah15** [comment]
@scshiv29-dev - Stack and Queue in Java
@osomat123 -  Circular Singly Linked List and Circular Doubly Linked List in C
@Rahulraj31 - Circular Doubly Linked List in C++
@aditik03 - Stack and Queue in C
@mukulpathania96 - Singly LL and Doubly LL in Java.
@APOORVANAGAR-2002 - Queue in C++
@AnanyaNagar -  Circular Singly LL in C++

Based on preferences received so far you all will be assigned these.
Please reach out in case of any issue.
Thank you!

**Comment by mukulpathania96** [comment]
Hey can you assign me the Circular Singly LL and Cicular DoublyLL?

**Comment by aditik03** [comment]
> Hey can you assign me the Circular Singly LL and Circular DoublyLL?

which language?

**Comment by sukritishah15** [comment]
> Hey can you assign me the Circular Singly LL and Cicular DoublyLL?

Yes, assigned both Circular Singly LL and Cicular Doubly LL in Java
@mukulpathania96

**Comment by Hasaber8** [comment]
https://github.com/sukritishah15/DS-Algo-Point/pull/424
I have already submitted a PR for a stack implementation example about infix and postfix in cpp , if you could assign it to me it will be great!

**Comment by Ritika0126** [comment]
I would like to add stack implementation in JavaScript, can you assign it to me.

**Comment by sukritishah15** [comment]
@Ritika0126 - JavaScript assigned
